### PR TITLE
Laterality - FHIR-32340

### PR DESCRIPTION
**Pull request by saulakravitz**
Resolution for FHIR-32340

Split STU2 bodylocationQualifier into two extensions:
a laterality qualifier (a la STU1) and a bodylocationQualifier.

LateralityVS would include: (as per STU1)
   - left
   - right
   - right and left
   - midline
   - unilateral

The bodyLocationQualiferVS would include:

Include codes from http://snomed.info/sct where concept is-a 106233006 (Topographical modifier)
Include codes from http://snomed.info/sct where concept is-a 272424004 (Relative sites)
excluding the codes in the LateralityVS that are under topgraphical modifier.

The same approach will be taken for Radiotherapy Treatment Location Qualifier, but the 
Treatment LocationQualifierVS would include a much smaller set of codes:
TreatmentLocationQualifierVS would include:
   -  Neck and Axillary Lymph Node Qualifiers
   - adjacent and non-adjacent
   - superior and inferior
   - posterior and anterior
   - upper, middle and lower
   - distal and proximal
   - entire and partial
   - peripheral and central
   - core
   - central

### CODE REVIEW

File: input/fsh/EX_Basic.fsh [hunk 1]
```diff
@@ -216,9 +216,10 @@ Description: "Anatomic volume for John Anyperson's brachytherapy."
 * patient = Reference(cancer-patient-john-anyperson)
 * description = "Structure of lower lobe of left lung"
 * location = SCT#31094006  "Structure of lobe of lung (body structure)"
-* locationQualifier[0] = SCT#7771000 "Left (qualifier value)"
-* locationQualifier[1] = SCT#261122009 "Lower (qualifier value)"
-
+// * locationQualifier[0] = SCT#7771000 "Left (qualifier value)"
+// * locationQualifier[1] = SCT#261122009 "Lower (qualifier value)"
```
[the remark this answers is not in the document]
Comment (saulakravitz) [reply input/fsh/EX_Basic.fsh:220]: done

**Comment by saulakravitz** [comment]
> `TreatmentIntentVS` is a broken link on https://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/laterality/group-treatment.html -- not sure what change caused this, but it looks ok on the STU2 ballot build.
> 
> image

TreatmentIntent-->ProcedureIntent

File: input/fsh/SD_TumorSize.fsh [hunk 2]
```diff
@@ -119,7 +119,8 @@ Description: "Represents a tumor after it has been removed from the body. Prior
 * identifier contains tumorIdentifier 0..*
 * identifier[tumorIdentifier] only BodyStructureIdentifier
 * identifier[tumorIdentifier] ^short = "Identifier to associate this specimen with a specific Tumor"
-* identifier[tumorIdentifier] ^definition = "To associate this with a specific BodyStructure conforming to the Tumor profile, add an identifier with a value that matches a persistent identifier from `BodyStructure.identifier.value` that is unique in the context of the Patient."
+// FHIR-32352
+// * identifier[tumorIdentifier] ^definition = "To associate this with a specific BodyStructure conforming to the Tumor profile, add an identifier with a value that matches a persistent identifier from `BodyStructure.identifier.value` that is unique in the context of the Patient."
```
Comment (markkramerus) [inline input/fsh/SD_TumorSize.fsh:123]: You should be updating and improving the definition, rather than eliminating it (according to FHIR-32352)

**Review by markkramerus — CHANGES_REQUESTED** [review]
Changes requested.

File: input/fsh/VS_Other.fsh [hunk 3]
```diff
@@ -44,11 +44,23 @@ Description: "Includes surgical procedure codes from SNOMED CT, ICD-10-PCS and C
...
+ValueSet: LateralityQualifierVS
+Id: mcode-laterality-qualifier-vs
+Title: "Laterality Qualifier Value Set"
+Description: "Qualifiers to specify laterality."
+* ^copyright = "This value set includes content from SNOMED CT, which is copyright © 2002+ International Health Terminology Standards Development Organisation (IHTSDO), and distributed by agreement between IHTSDO and HL7. Implementer use of SNOMED CT is not covered by this agreement"
+* SCT#51440002	"Right and left (qualifier value)"
+* SCT#399488007	"Midline (qualifier value)"	
+* SCT#24028007	"Right (qualifier value)"	
+* SCT#7771000	"Left (qualifier value)"	
+* SCT#66459002	"Unilateral (qualifier value)"
```
[the remark this answers is not in the document]
Comment (saulakravitz) [reply input/fsh/VS_Other.fsh:63]: Week case for unilateral (from SharonS): "Unilateral is murkier. Wendy admitted it is a catch-all used by CTRs when laterality is not specified in the chart but laterality is definitely only L or R but they don’t know which one. She gave the example of a kidney. People only have 2 kidneys - a L and a R kidney so the closest they can get to laterality is “unilateral”. May was on the call and participated in the conversation. She was trying to recall why it was added to the VS originally. She will share additional info with the internal MITRE group when you meet next week. There doesn’t seem to be a very strong argument for unilateral."

File: input/pagecontent/StructureDefinition-mcode-comorbidities-elixhauser-intro.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ If a comorbidity is present, the user can optionally supply a specific code deta
 
 For any component, "present" or "absent" refers back to a comorbidity category defined by `Observation.component`.code. Since there are no existing codes for individual Elixhauser categories, mCODE provides them. For example, a component relating to congestive heart failure (CHF) will have `component.code.coding.system` of `http://hl7.org/fhir/us/mcode/CodeSystem/mcode-elixhauser-category-cs` and `component.code.coding.code` "CHF". The implication is that the patient has a disease found in the *specific version* of Elixhauser CHF codes corresponding to the definition of the "CHF" code; in this case, one of the codes in [Elixhauser Comorbid Condition Value Set for Congestive Heart Failure](ValueSet-elixhauser-congestive-heart-failure-vs.html).
 
-If an overall score is calculated, it should be recorded in Observation.value[valueInteger].   This is not required.
+If an overall score is calculated, it should be recorded in Observation.valueInteger. The score is not required.
```
[the remark this answers is not in the document]
Comment (saulakravitz) [reply input/pagecontent/StructureDefinition-mcode-comorbidities-elixhauser-intro.md:25]: That is in a separate PR.

File: input/fsh/SD_TumorSize.fsh (hunk 2 above)
Comment (saulakravitz) [reply input/fsh/SD_TumorSize.fsh:123]: One of the criticisms was using the definition for instructions on usage.   This was moved to the usage of Tumor Specimen -- https://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/laterality/StructureDefinition-mcode-tumor-specimen.html

File: input/fsh/EX_Basic.fsh [hunk 5]
```diff
@@ -216,9 +216,8 @@ Description: "Anatomic volume for John Anyperson's brachytherapy."
 * patient = Reference(cancer-patient-john-anyperson)
 * description = "Structure of lower lobe of left lung"
 * location = SCT#31094006  "Structure of lobe of lung (body structure)"
-* locationQualifier[0] = SCT#7771000 "Left (qualifier value)"
-* locationQualifier[1] = SCT#261122009 "Lower (qualifier value)"
-
+* location.extension[locationQualifier][0].valueCodeableConcept = SCT#7771000 "Left (qualifier value)"
```
[the remark this answers is not in the document]
Comment (saulakravitz) [reply input/fsh/EX_Basic.fsh]: Good catch.

File: input/fsh/VS_Other.fsh (hunk 3 above)
[the remark this answers is not in the document]
Comment (saulakravitz) [reply input/fsh/VS_Other.fsh]: Dropped unilateral.

**Pull request merged**